src/App.tsx

```diff
@@ -17,18 +57,36 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                query={query}
+                status={status}
+                onInputQuery={inputQuery}
+                onSelectStatus={selectStatus}
+              />
             </div>
 
             <div className="block">
-              <Loader />
-              <TodoList />
+              {todos.length === 0
```

Comment:
Let's add some boolean flag `isLoading` and use it here.
You can create boolean useState with initial value `true` and change it to `false` after you get a response from the server